packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -730,6 +1022,36 @@ export const init =
 
             const providerResult = await dispatch(initProvider());
             if (!providerResult) {
+                dispatch(abortInit());
+                return false;
+            }
+        }
+
+        // 4. get the metadata minimum version to determine a migration is needed
+        const metadataFilesList = await dispatch(getMetadataFilesList());
+        const metadataFilesMimimumVersion = getMetadataFilesMinimumVersion(metadataFilesList);
+
+        if (getState().metadata.initiating && !metadataFilesMimimumVersion) {
```

Comment:
`!metadataFilesMimimumVersion` check seems to be useless since `getMetadataFilesMinimumVersion` always returns number

Reply:
done